Example/EmissionTests/TestHelper.m

```diff
@@ -0,0 +1,89 @@
+#import "TestHelper.h"
+
+#import <SDWebImage/SDImageCache.h>
+#import <SDWebImage/SDWebImageManager.h>
+
+@interface TestHelper ()
+@property (nonatomic, strong, readwrite) UIWindow *window;
+@property (nonatomic, strong, readwrite) NSArray<NSDictionary *> *artworksPages;
+@property (nonatomic, strong, readwrite) NSArray<NSDictionary *> *artworks;
+@end
+
+@implementation TestHelper
+
++ (instancetype)sharedHelper;
+{
+  return (TestHelper *)UIApplication.sharedApplication.delegate;
+}
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions;
+{
+  [self assertCorrectEnvironment];
```

Comment:
This whole diff is a bit over my head - it seems like a lot of Cocoa-wrangling. Should I make a concerted effort to understand everything that's going on or can I focus more on the RN side of things?